sonar-db/src/main/resources/org/sonar/db/component/ComponentMapper.xml

```diff
@@ -250,6 +250,40 @@
     ORDER BY UPPER(p.name), p.name
   </select>
 
+  <select id="selectByQuery" resultType="Component">
+    select
+    <include refid="componentColumns"/>
+    <include refid="sqlSelectByQuery" />
+    ORDER BY LOWER(p.name), p.name
```

Comment:
it's done elsewhere, and it permits to have a non case sensitive order consistent on all db